package.json

```diff
@@ -58,7 +58,6 @@
 		"start": "npm run develop",
 		"develop": "grunt dev --force",
 		"api-docs": "jsdoc --configure .jsdoc.json",
-		"imports-gen": "node ./build/imports-generator",
```

Comment:
We need to move work out of grunt, not move more stuff into it.

Reply:
If you look at the changes here - https://github.com/dequelabs/axe-core/pull/1388/files#diff-35b4a816e0441e6a375cd925af50752cR334, all we have done is invoke `node` instead of a `npm` command via an already used plugin `grunt-run`, so nothing has been moved around per se.

Reply:
Perhaps we could do the inverse and not invoke this via `grunt-run` but as an npm script instead.